languages/python/oso/tests/test_polar_roles.py

```diff
@@ -309,88 +297,9 @@ def test_duplicate_resource_name(init_oso):
     oso.load_str(policy)
 
     with pytest.raises(OsoError):
-        oso.roles.synchronize_data()
-
-
-@pytest.mark.skip("TODO: validation")
-def test_nested_dot_relationship(init_oso):
-    # - multiple dot lookups throws an error for now
-    oso, session = init_oso
-    policy = """
-    resource(_type: Org, "org", actions, roles) if
-        actions = ["invite"] and
-        roles = {
-            member: {
-                permissions: ["invite"]
-            }
-        };
-
-    resource(_type: Issue, "issue", actions, roles) if
-        actions = [
-            "edit"
-        ];
-
-    child_parent(issue, parent_org) if
-        issue.repo.org = parent_org;
-
-    allow(actor, action, resource) if
-        role_allows(actor, action, resource);
-    """
-    oso.load_str(policy)
-
-    with pytest.raises(OsoError):
-        oso.roles.synchronize_data()
-
-
-@pytest.mark.skip("TODO: validation")
-def test_bad_relationship_lookup(init_oso):
-    # - nonexistent attribute lookup throws an error for now
-    oso, session = init_oso
-    policy = """
-    resource(_type: Org, "org", actions, roles) if
-        actions = ["invite"] and
-        roles = {
-            member: {
-                permissions: ["invite"]
-            }
-        };
-
-    resource(_type: Repo, "repo", actions, {}) if
-        actions = [
-            "pull"
-        ];
-
-    child_parent(repo: Repo, parent_org) if
-        # INCORRECT FIELD NAME
-        repo.organization = parent_org and
-        parent_org matches Org;
-    """
-    oso.load_str(policy)
-
-    with pytest.raises(OsoError):
-        oso.roles.synchronize_data()
```

Comment:
I think we should keep this test around (skipped) b/c this is a validation I would love to try to do in the future. Probably wouldn't ever work for Python since we can't generically walk relationship fields, but I could imagine it working for other host langs / frameworks.